app-springboot/src/main/java/org/keycloak/quickstart/springboot/config/SecurityConfig.java

```diff
@@ -5,8 +5,7 @@
  */
 package org.keycloak.quickstart.springboot.config;
```

Comment:
Why are these changes required? This is indicating breaking changes in our adapters.